WIP - Perform static build on windows

SylvainCorlay opened the pull request:
This appears to be required to build openexr on windows.

SylvainCorlay committed:
Perform static build on windows

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

SylvainCorlay committed:
Perform static build on windows

SylvainCorlay commented:
Before that change, openexr cannot find the static libraries on windows.
After that change, building the openexr conda recipe yields

```
CMake Error in OpenEXR/IlmImf/CMakeLists.txt:
  Imported target "IlmBase::Half_static" includes non-existent path

    "/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in OpenEXR/IlmImf/CMakeLists.txt:
  Imported target "IlmBase::Half_static" includes non-existent path

    "/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in OpenEXR/IlmImf/CMakeLists.txt:
  Imported target "IlmBase::Half_static" includes non-existent path

    "/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in OpenEXR/IlmImf/CMakeLists.txt:
  Imported target "IlmBase::Half_static" includes non-existent path

    "/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in OpenEXR/IlmImf/CMakeLists.txt:
  Imported target "IlmBase::Iex" includes non-existent path

    "/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in OpenEXR/IlmImf/CMakeLists.txt:
  Imported target "IlmBase::Iex" includes non-existent path

    "/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in OpenEXR/IlmImfUtil/CMakeLists.txt:
  Imported target "IlmBase::Iex" includes non-existent path

    "/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in OpenEXR/IlmImfUtil/CMakeLists.txt:
  Imported target "IlmBase::Iex" includes non-existent path

    "/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.
```
